codecparsers/h265Parser.cpp

```diff
@@ -24,6 +24,38 @@
 #include <math.h>
 #include <stddef.h> //offsetof
 
+#define READ(x, f)                          \
```

Comment:
They do not need know br, nr the use, the it's duplicate work to write READ_BITS(nr, ....), if you change this to h264 style, The reader only need focus what fields he read, the do not need pay attention to nr or br. Other point is nal reader is subclass of bit reader, name it to br is no problem;)